src/pages/resultsView/plots/PlotsTabUtils.tsx

```diff
@@ -178,7 +178,7 @@ export const WATERFALLPLOT_SIDELENGTH_SAMPLE_MULTIPLICATION_FACTOR = 1.6;
 export interface IAxisData {
     data: {
         uniqueSampleKey: string;
-        value: string | number | (string[]) | (number[]); // if theres a list, then we'll make one data point per value
```

Comment:
[Don't require parens for same-operator logical expressions](https://prettier.io/blog/2019/11/09/1.19.0.html#dont-require-parens-for-same-operator-logical-expressions-6864-by-jridgewell)